files/build_templates/buffers_config.j2

```diff
@@ -57,6 +57,8 @@ def
                          {%- set roles2 = neighbor_role + '_' + switch_role %}
                          {%- set roles1 = roles1 | lower %}
                          {%- set roles2 = roles2 | lower %}
+                         {%- set roles1 = roles1.replace('backend', '') %}
```

Comment:
So its no-op if it is regular torrouter, correct?

Reply:
correct, will just return the original string if 'backend' is not present.